Sources/SecureStore/KeyManagerService.swift

```diff
@@ -41,8 +41,8 @@ extension KeyManagerService {
         let tag = name.data(using: .utf8) else { return }
 
         let attributes: NSDictionary = [
-            kSecAttrKeyType: kSecAttrKeyTypeECSECPrimeRandom,
-            kSecAttrKeySizeInBits: 256,
+            kSecAttrKeyType: kSecAttrKeyTypeRSA,
+            kSecAttrKeySizeInBits: 4096,
```

Comment:
are we able to try and figure out a way to unit test this please?
ideally want to ensure we don't accidentally make any unintended changes in this part of the codebase